src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -429,10 +427,16 @@ class LicenceService(
   }
 
   @Transactional
-  fun inActivateLicences(licenceIds: List<Long>) {
+  fun activateLicencesByIds(licenceIds: List<Long>) {
     val matchingLicences =
-      licenceRepository.findAllById(licenceIds).filter { licence -> licence.statusCode == APPROVED }
-    val inActivatedLicences = matchingLicences.map { licence -> licence.copy(statusCode = INACTIVE) }
+      licenceRepository.findAllById(licenceIds).filter { it.statusCode == APPROVED }
+    if (matchingLicences.any()) {
```

Comment:
No point checking here, the lower level will ensure what to do if there are no licences.

Reply:
True, it just prevents an unnecessary method call if there's nothing that matches.
I can remove it if you'd prefer though!

Reply:
think it'd make sense to, makes the code a bit simpler, and reduces need for extra tests.